### PR TITLE
Added "Todos" option and Filter logic

### PR DESCRIPTION
**Pull request by dbuendia**
- Added "Todos" option for all selects.
- Added filter logic (only selected elements from dataset are displayed)

Captura de Pantalla 2022-08-08 a las 15 40 37
.

### CODE REVIEW

File: src/components/Hotels.jsx [hunk 1]
```diff
@@ -1,19 +1,54 @@
...
+      } else {
+        size = "Pequeño";
+      }
+      return size === capacity;
+    });
+
   return (
     <div className="hotels">
-      {hotelList.map((hotel) => {
+      {filteredHotels.map((elem) => {
```
Comment (havebeeair) [inline src/components/Hotels.jsx:42]: Why changed to `elem`? I think `hotel` suits best.

File: src/components/Hotels.jsx [hunk 2]
```diff
@@ -1,19 +1,54 @@
...
+    })
+    .filter((hotel) => {
+      let size = "";
+
+      if (capacity === "Todos") {
+        return true;
+      }
+
+      if (hotel.rooms > 20) {
+        size = "Grande";
```
Comment (havebeeair) [inline src/components/Hotels.jsx:31]: Can we reference the variables from `consts.js` instead of hardcoding?

**Review by havebeeair — CHANGES_REQUESTED** [review]

File: src/components/Hotels.jsx (hunk 1 above)
Comment (dbuendia) [reply src/components/Hotels.jsx:43]: As we discussed privately, elem will remain at least for now. 👍🏻

File: src/components/Hotels.jsx (hunk 2 above)
Comment (dbuendia) [reply src/components/Hotels.jsx]: Updated!

**Pull request merged**